src/Microsoft.AspNet.Mvc.Core/Rendering/IHtmlHelper.cs

```diff
@@ -46,6 +47,21 @@ public interface IHtmlHelper
         ViewDataDictionary ViewData { get; }
 
         /// <summary>
+        /// Gets the <see cref="IHtmlEncoder"/> to be used for encoding Html.
+        /// </summary>
+        IHtmlEncoder HtmlEncoder { get; }
+
+        /// <summary>
+        /// Gets the <see cref="IUrlEncoder"/> to be used for encoding Url.
```

Comment:
Take a look at the doc comments on the IXyzEncoder classes themselves if you need inspiration.